pkg/kernels/kernels.go

```diff
@@ -151,3 +151,14 @@ func GenericKprobeObjs() (string, string) {
 		return "bpf_generic_kprobe.o", "bpf_generic_retkprobe.o"
 	}
 }
+
+// ExecObj returns the exec object based on the kernel version
+func ExecObj() string {
```

Comment:
nice, I would probably keep this one in the base.go file, but feel free to ignore ;-)

Reply:
makes sense. moved it to base.go :)